docs/administration/configuring-replication/create-replication-endpoints.md

```diff
@@ -7,9 +7,9 @@ To replicate image repositories from one instance of Harbor to another Harbor or
 
 1. Go to **Registries** and click the **+ New Endpoint** button.
 
-   ![New replication endpoint](../../../img/replication-endpoint1.png)
+   ![New replication endpoint](../../img/replication-endpoint1.png)
 
-1. For **Provider**, use the drop-down menu to select the type of registry to set up as a replication endpoint.
+2. For **Provider**, use the drop-down menu to select the type of registry to set up as a replication endpoint.
```

Comment:
We want to keep the numbered lists as 1., as moving things around or adding a new step in the middle of the structure doesn't require any reordering of the numbers then. See https://github.com/DavidAnson/markdownlint/blob/master/doc/Rules.md#md029.